### PR TITLE
Select-inverse of current selection via Cmd/crtl-I

### PR DESCRIPTION
**Pull request by thirdknife**
This PR add's the functionality.

### CODE REVIEW

File: app/actions/selectInverse.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default function selectInverse(input, tree, output) {
+    //compare the sequenceString being pasted in with what's already stored in the clipboard
+    var sequenceLength = tree.get(['sequenceLength']);
```
Comment (tnrich) [inline app/actions/selectInverse.js]: it looks like we don't actually need the sequence length

**Comment by tnrich** [comment]
Looks good other than the sequence length stuff I commented about above :) Awesome work! You're gonna need some new issues to tackle, huh?

File: app/actions/selectInverse.js (hunk 1 above)
Comment (thirdknife) [inline app/actions/selectInverse.js]: yeah! I didn't even look at it. Sending changes.

**Comment by thirdknife** [comment]
@tnrich yes I do.

**Pull request merged**